README.md

```diff
@@ -40,494 +40,196 @@ Ensure packages are installed with correct version numbers by running:
   npm install --save react-dates moment@>=#.## react@>=#.## react-dom@>=#.## react-addons-shallow-compare@>=#.##
   ```
 
-#### Include component
+### Include component
 ```js
-import { SingleDatePicker } from 'react-dates';
+import { DateRangePicker, SingleDatePicker, DayPickerRangeController } from 'react-dates';
 ```
 
-#### Include CSS
+### Include CSS
 When you use Webpack with CSS loader:
 ```js
 import 'react-dates/lib/css/_datepicker.css';
 ```
 Otherwise create a CSS file with the contents of `require.resolve('react-dates/lib/css/_datepicker.css')` and include it in the html `<head>` section.
```

Comment:
What do you mean by needs to be compiled? `react-dates/lib/css/_datepicker.css` has already been run through a SASS compiler.